cypress/e2e/cloud/operator.test.ts

```diff
@@ -49,10 +49,13 @@ describe('Operator Page', () => {
       .invoke('attr', 'placeholder')
       .should('contain', 'Filter accounts by email')
 
-    cy.getByTestID('operator-resource--searchbar').type('asalem', {
-      force: true,
-      delay: 50,
-    })
+    cy.focused()
+    cy.getByTestID('operator-resource--searchbar')
+      .scrollIntoView()
```

Comment:
will this fix it? in the failed test, what was typed was `alm`. it's not typing certain characters.